awx_collection/plugins/modules/tower_inventory_source.py

```diff
@@ -20,333 +20,347 @@
-          local child hosts and groups not found on the external source will
-          remain untouched by the inventory update process.
+        - Delete child groups and hosts not found in source.
       type: bool
+      default: 'no'
     overwrite_vars:
       description:
-        - >-
-          If set, all variables for child groups and hosts will be removed
-          and replaced by those found on the external source. When not checked,
-          a merge will be performed, combining local variables with those found
-          on the external source.
+        - Override vars in child groups and hosts with those from external source.
       type: bool
+    custom_virtualenv:
+      version_added: "2.9"
+      description:
+        - Local absolute file path containing a custom Python virtualenv to use.
+      type: str
+      required: False
+      default: ''
+    timeout:
+      description: The amount of time (in seconds) to run before the task is canceled.
+      type: int
+    verbosity:
+      description: The verbosity level to run this inventory source under.
+      type: int
+      choices: [ 0, 1, 2 ]
     update_on_launch:
       description:
-        - >-
-          Each time a job runs using this inventory, refresh the inventory from
-          the selected source before executing job tasks.
+        - Refresh inventory data from its source each time a job is run.
       type: bool
+      default: 'no'
     update_cache_timeout:
       description:
-        - >-
-          Time in seconds to consider an inventory sync to be current. During
-          job runs and callbacks the task system will evaluate the timestamp of
-          the latest sync. If it is older than Cache Timeout, it is not
-          considered current, and a new inventory sync will be performed.
+        - Time in seconds to consider an inventory sync to be current.
       type: int
+    source_project:
+      description:
+        - Project to use as source with scm option
+      type: str
+    update_on_project_update:
+      description: Update this source when the related project updates if source is C(scm)
+      type: bool
     state:
       description:
         - Desired state of the resource.
       default: "present"
       choices: ["present", "absent"]
       type: str
+    tower_oauthtoken:
+      description:
+        - The Tower OAuth token to use.
+      required: False
+      type: str
 extends_documentation_fragment: awx.awx.auth
 '''
 
-
 EXAMPLES = '''
-- name: Add tower inventory source
-  tower_inventory_source:
-    name: Inventory source
-    description: My Inventory source
-    inventory: My inventory
-    organization: My organization
-    credential: Devstack_credential
-    source: openstack
-    update_on_launch: true
-    overwrite: true
-    source_vars: '{ private: false }'
+- name: Add tower group
+  tower_group:
+    name: localhost
+    description: "Local Host Group"
+    inventory: "Local Inventory"
```

Comment:
What's going on with this example? This is the `tower_inventory_source` module, so the example should be about that and not `tower_group`.

Reply:
Woops!  Will update that example.